spec/requests/admin/users_manage_spec.rb

```diff
@@ -68,7 +68,7 @@ def full_profile
     end
 
     it "deletes duplicate user" do
-      post "/admin/users/#{user.id}/merge", params: { user: { merge_user_id: user2.id } }
```

Comment:
These paths also stay the same with the restructure, but whilst I was here I just moved them over to path helpers